test/Microsoft.Framework.Caching.Memory.Tests/TimeExpirationTests.cs

```diff
@@ -46,18 +46,14 @@ public void AbsoluteExpirationExpires()
             var clock = new TestClock();
             var cache = CreateCache(clock);
             var key = "myKey";
-            var obj = new object();
+            var value = new object();
 
-            var result = cache.Set(key, context =>
-            {
-                context.SetAbsoluteExpiration(clock.UtcNow + TimeSpan.FromMinutes(1));
```

Comment:
I think this was explicitly testing the other overload of SetAbsoluteExpiration.